sky/serve/autoscalers.py

```diff
@@ -64,10 +64,25 @@ def __init__(self, spec: 'service_spec.SkyServiceSpec') -> None:
             max_replicas: Maximum number of replicas. Default to fixed
                 number of replicas, i.e. min_replicas == max_replicas.
             target_num_replicas: Target number of replicas output by autoscaler.
+            latest_version: latest version of the service.
         """
         self.min_replicas: int = spec.min_replicas
-        self.max_replicas: int = spec.max_replicas or spec.min_replicas
+        self.max_replicas: int = (spec.max_replicas if spec.max_replicas
+                                  is not None else spec.min_replicas)
+        # Target number of replicas is initialized to min replicas.
         self.target_num_replicas: int = spec.min_replicas
+        self.latest_version: int = constants.INITIAL_VERSION
+
+    def update_version(self, version: int,
+                       spec: 'service_spec.SkyServiceSpec') -> None:
+
+        self.latest_version = version
```

Comment:
What if the `version <= self.latest_version`, should we only set the latest version when `version` is larger?